scripts/setup_zipkin.sh

```diff
@@ -22,9 +22,12 @@
 
 #!/bin/bash
 
+set -e
 # install zipkin pods
 kubectl apply -f https://raw.githubusercontent.com/istio/istio/release-1.9/samples/addons/extras/zipkin.yaml
 
+sleep 10s
+
 # enable tracing in Knative
 kubectl patch configmap/config-tracing \
   -n knative-serving \
```

Comment:
I guess this is the name of the namespace, right? probably, you were missing info in the traces before because some of the services were running in different namespaces. What do you think?

Reply:
this namespace is for the config file location. Namespaces for the zipki pods are set by the config file provided by the istio team (line number 27).